curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-fetch-and-promises-by-building-an-fcc-authors-page/63ff7f1fa497db1ef87bac1d.md

```diff
@@ -0,0 +1,167 @@
+---
+id: 63ff7f1fa497db1ef87bac1d
+title: Step 9
+challengeType: 0
+dashedName: step-9
+---
+
+# --description--
+
+Now, we need to take what we need from each of the author object. This is called destructuring. Inside each object in the array are `author`, `image`, `url`, and `bio` items. We will need all of them to populate the author info on the UI.
+
+This is how you can destructure in JavaScript:
+
+```js
+const freeCodeCamp = {
+  founder: 'Quincy Larson',
+  type: 'Non-profit',
+  funding: 'Donations',
+};
+
+const { founder, type } = freeCodeCamp;
+
+console.log(founder) // Quincy Larson
+console.log(type) // Non-profit
+```
+
+That means I took out the `founder` and `type` items from the object.
```

Comment:
While this is a good explanation, I don't think we need to give that much detail at this point in the curriculum because campers will have worked with object destructing a few times in earlier projects.

Personally, I think we should remove this and just leave the last two paragraphs you wrote.